test/deps.test.ts

```diff
@@ -1,7 +1,7 @@
 export {
     readAll,
     readerFromIterable,
-} from "https://deno.land/std@0.178.0/streams/mod.ts";
+} from "https://deno.land/std@0.181.0/streams/mod.ts";
```

Comment:
```suggestion
} from "https://deno.land/std@0.184.0/streams/mod.ts";
```

in other places too.